lib/src/main/java/io/ably/lib/util/AblyErrors.java

```diff
@@ -0,0 +1,169 @@
+package io.ably.lib.util;
+
+public enum AblyErrors {
```

Comment:
This can be modified same as https://github.com/ably/ably-java/pull/881#discussion_r1059132748. 
Create a separate static class 
```
    // Also, more helpful in IDE code indexing/lookups
    public static class AblyErrorCode {
        public static final int NO_ERROR = 10000;
        public static final int BAD_REQUEST= 40000;
        ......
    }  
```

This is also more readable.
```
System.out.println(AblyErrorCode.BAD_REQUEST);
```

I think the message is almost the same for most of the string constants, so there's no point in keeping it. 
If you want to add a description to some of those messages, you can write an extra static method

```
 public static class AblyErrorCode {
        public static final int NO_ERROR = 10000;
        public static final int BAD_REQUEST= 40000;
        ......
        
      static string describe(int errorCode) {
       switch(errorCode) {
         case NO_ERROR:
              return "no error";
           }
        }
    }
```
Usage

```
System.out.println(AblyErrorCode.describe(AblyErrorCode.BAD_REQUEST));
```

The main aim is to simplify the readability of constants. Enum just makes it difficult to use at places and doesn't fulfill the purpose.

Reply:
Agreed, I will implement your suggestions.

Reply:
Thanks!